tesseract_core/runtime/cli.py

```diff
@@ -491,6 +493,32 @@ def openapi_schema() -> dict:
         _create_user_defined_cli_command(app, func, out_stream)
 
 
+def _configure_required_file_load() -> None:
+    skip_required_file_load_args = [
+        "-h",
+        "--help",
+        "--version",
+        "-v",
+        "openapi-schema",
+        "check",
+    ]
+    if "--input-path" in sys.argv:
+        # Make sure input_path is available when loading tesseract_api.py for the first time
+        # (needed to resolve required files)
+        input_path_index = sys.argv.index("--input-path")
+        if len(sys.argv) <= input_path_index + 1:
+            raise ValueError("No input path provided after --input-path argument.")
+        input_path = Path(sys.argv[input_path_index + 1])
+        update_config(input_path=input_path.as_posix())
+    if any(arg in sys.argv for arg in skip_required_file_load_args):
+        # Skip loading if unnecessary, based on above arguments
+        os.environ["_TESSERACT_IS_BUILDING"] = "1"
+        import importlib
+
+        # Need to reload to ensure that experimental.IS_BUILDING has correct value
+        importlib.reload(experimental)
```

Comment:
No need to set the envvar, just update the attribute directly:

```suggestion
        tesseract_core.runtime.experimental.IS_BUILDING = True
```